_sass/index.scss

```diff
@@ -238,10 +238,47 @@ body {
 }
 
 .metrocommon {
-  background-color: rgb(70, 119, 72);
-  color: white;
-  width: 95%;
-  height: 30rem;
+  background: $color_light_gold;
+  display: flex;
+  font-family: $font_family-primary;
+  height: 40rem;
+  justify-content: center;
+
+  &__left {
+    background: url('images/metrocommon-triangles-yellow.png') no-repeat;
```

Comment:
The styling is solid, but I have a question about the intention of the triangles. In the XD, the lefthand column is a bit smaller and shorter, but there's also less blank space than we anticipated. On my widescreen monitor, it looks like this:

![Screen Shot 2019-12-09 at 8 36 25 AM](https://user-images.githubusercontent.com/10645500/70440056-5811d380-1a5f-11ea-8e41-a6bcc38d97dc.png)

Particularly, the bottom left triangle looks out of place to me since it doesn't have a leftmost edge. Is the goal to have two different sizes of triangles (in which case, keep as is) or is the goal to make the two columns fit in a single screen size?

Reply:
Waiting for new image from @theryankelly to change this per discussion last week/this morning.

Reply:
Forgot that we discussed this last week--my bad!